themes/src/main/resources-community/theme/base/account/messages/messages_es.properties

```diff
@@ -144,4 +201,158 @@ invalidPasswordMinUpperCaseCharsMessage=Contrase\u00F1a incorrecta: debe contene
+authenticatorActionSetup=Configurar 
+authenticatorSMSTitle=C\u00F3digo SMS
+authenticatorSMSMessage=Keycloak enviar\u00E1 el c\u00F3digo de verificaci\u00F3n a su tel\u00E9fono como la autenticaci\u00F3n de dos factores.
+authenticatorSMSFinishSetUpMessage=Los mensajes de texto se env\u00EDan a
+authenticatorDefaultStatus=Defecto
+authenticatorChangePhone=Cambiar n\u00FAmero de tel\u00E9fono
+authenticatorBackupCodesTitle=C\u00F3digos de respaldo
+authenticatorBackupCodesMessage=Obtenga sus c\u00F3digos de respaldo de 8 d\u00EDgitos
+authenticatorBackupCodesFinishSetUpMessage=Se generaron 12 c\u00F3digos de respaldo en este momento. Cada uno se puede usar una vez.
+
+#Authenticator - Mobile Authenticator setup
+authenticatorMobileSetupTitle=Configuraci\u00F3n del autenticador m\u00F3vil
+smscodeIntroMessage=Ingrese su n\u00FAmero de tel\u00E9fono y se le enviar\u00E1 un c\u00F3digo de verificaci\u00F3n.
+mobileSetupStep1=Instale una aplicaci\u00F3n de autenticaci\u00F3n en su tel\u00E9fono. Las aplicaciones enumeradas aqu\u00ED son compatibles.
+mobileSetupStep2=Abra la aplicaci\u00F3n y escanee el c\u00F3digo QR:
+mobileSetupStep3=Ingrese el c\u00F3digo de un solo uso proporcionado por la aplicaci\u00F3n y haga clic en Guardar para finalizar la configuraci\u00F3n.
+scanBarCode=\u00BFQuieres escanear el c\u00F3digo de QR?
+enterBarCode=Ingrese el c\u00F3digo de un solo uso
+doCopy=Copiar
+doFinish=Finalizar
+
+#Authenticator - SMS Code setup
+authenticatorSMSCodeSetupTitle=Configuraci\u00F3n de c\u00F3digo de SMS
+chooseYourCountry=Elige tu pa\u00EDs
+enterYourPhoneNumber=Ingrese su n\u00FAmero telef\u00F3nico
+sendVerficationCode=Enviar el c\u00F3digo de verificaci\u00F3n
+enterYourVerficationCode=Ingrese su c\u00F3digo de verificaci\u00F3n
+
+#Authenticator - backup Code setup
+authenticatorBackupCodesSetupTitle=Configuraci\u00F3n de c\u00F3digos de respaldo
+backupcodesIntroMessage=Si pierde el acceso a su tel\u00E9fono, a\u00FAn puede iniciar sesi\u00F3n en su cuenta a trav\u00E9s de c\u00F3digos de respaldo. Gu\u00E1rdelos en un lugar seguro y accesible.
+realmName=Dominio
+doDownload=Descargar
+doPrint=Imprimir
+backupCodesTips-1=Cada c\u00F3digo de respaldo se puede usar una vez.
+backupCodesTips-2=Estos c\u00F3digos fueron generados en
+generateNewBackupCodes=Generar nuevos c\u00F3digos de respaldo
+backupCodesTips-3=Cuando genera nuevos c\u00F3digos de respaldo, los c\u00F3digos actuales ya no funcionar\u00E1n.
+backtoAuthenticatorPage=Volver a la p\u00E1gina del autenticador
+
+
+#Resources
+resources=Recursos
+sharedwithMe=Comparte conmigo
+share=Compartir
+sharedwith=Compartido con
+accessPermissions=Permisos de acceso
+permissionRequests=Solicitudes de permisos
+approve=Aprovar
+approveAll=Aprovar todo
+people=Personas
+perPage=Por p\u00E1gina
+currentPage=P\u00E1gina actual
+sharetheResource=Compartir el recurso
+group=Grupo
+selectPermission=Seleccionar permiso
+addPeople=Agregue personas para compartir su recurso con
```

Comment:
In this file "Añadir" is being used as translation for add, also the translation is too literal, it could be:
"Añada personas con quien compartir su recurso"